tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/browsing_context_name.html.ini

```diff
@@ -1,5 +1,3 @@
 [browsing_context_name.html]
   type: testharness
-  [Retaining window.name on history traversal]
-    expected: FAIL
-
+  disabled: see https://github.com/whatwg/html/issues/490
```

Comment:
The test looks for events for an iframe onload for each step. The initial frame has no src attribute, which means it will load about:blank. Based on whatwg/html#490, it looks like the behavior is unknown when the src changes before the onload event fires. That would make this test racy depending on the browser I'm guessing.